package.json

```diff
@@ -24,7 +24,6 @@
   },
   "homepage": "https://github.com/limechain/subsembly#readme",
   "dependencies": {
-    "@assemblyscript/loader": "^0.17.1",
```

Comment:
don't we need it anymore?

Reply:
yeah, we don't need it inside the root directory. It is used by cli and we have to add it to cli